lilypond.yaml

```diff
@@ -0,0 +1,76 @@
+---
+name: lilypond
+buildsystem: autotools
+config-opts:
+  - "--disable-documentation"
+sources:
+  - type: archive
+    url: http://lilypond.org/download/sources/v2.20/lilypond-2.20.0.tar.gz
+    sha256: 595901323fbc88d3039ca4bdbc2d8c5ce46b182edcb3ea9c0940eba849bba661
+build-options:
+  env:
+    PATH: /usr/lib/sdk/texlive/bin/x86_64-linux:/usr/lib/sdk/texlive/bin:/usr/bin:/app/bin
+
+modules:
+  # To be removed when next lilypond stable (2.22) is released.
+  - shared-modules/python2.7/python-2.7.json
+
+  - name: guile
+    buildsystem: autotools
+    build-options:
+      cflags: "-Wno-unused-but-set-variable -Wno-misleading-indentation
+              -Wno-stringop-truncation -Wno-deprecated-declarations"
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/guile/guile-1.8.8.tar.gz
+        sha256: c3471fed2e72e5b04ad133bbaaf16369e8360283679bcf19800bc1b381024050
+
+  - name: pango
+    sources:
+      - type: archive
+        url: https://ftp.gnome.org/pub/GNOME/sources/pango/1.42/pango-1.42.4.tar.xz
```

Comment:
is it for pangoXft that you need it?

Reply:
All I know is that [lilypond needs Pango](http://lilypond.org/doc/v2.21/Documentation/contributor/requirements-for-running-lilypond).